antenna/app.py

```diff
@@ -252,24 +260,31 @@ def get_resource_by_name(self, name):
         return self._all_resources[name]
 
     def get_resources(self):
-        """Returns a list of registered resources"""
+        """Return a list of registered resources."""
         return self._all_resources.values()
 
     def get_runtime_config(self, namespace=None):
+        """Return generator of runtime configuration for all resources."""
         for res in self.get_resources():
             if hasattr(res, 'get_runtime_config'):
                 for item in res.get_runtime_config(namespace):
                     yield item
 
+    def verify(self):
+        """Verify that Antenna is ready to start."""
+        self.hb.verify()
```

Comment:
The heartbeat system already had registration for hooks, so I decided to make verification part of the heartbeat stuff.